core/src/main/scala/pureconfig/configurable/package.scala

```diff
@@ -4,17 +4,31 @@ import scala.util.Try
 import java.time.{ LocalDate, LocalDateTime, LocalTime }
 import java.time.format.DateTimeFormatter
 
+/**
+ * Provides methods that create [[ConfigConvert]] instances from a set of parameters used to configure the instances.
+ *
+ * The result of calling one of the methods can be assigned to an `implicit val` so that `pureconfig` will be able to
+ * use it:
+ * {{{
+ *   implicit val localDateConfigConvert = makeLocalDateConfigConvert(DateTimeFormatter.ISO_TIME)
```

Comment:
I think it might be useful to try to explain where the user of the library needs to include this line of code, as is always the case with Scala's implicits.